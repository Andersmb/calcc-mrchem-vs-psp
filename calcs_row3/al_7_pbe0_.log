Starting job 5688394 on c1-[21,26],c2-39,c3-[13,27],c5-22 at Fri May 20 10:30:27 CEST 2022

Copying al_7_pbe0_.out to /cluster/home/ambr/projects/psp/row3/calcs/doublets
Copying al_7_pbe0_.json to /cluster/home/ambr/projects/psp/row3/calcs/doublets

Task and CPU usage stats:
JobID           JobName  AllocCPUS   NTasks     MinCPU MinCPUTask     AveCPU    Elapsed ExitCode 
------------ ---------- ---------- -------- ---------- ---------- ---------- ---------- -------- 
5688394      al_7_pbe0_        180                                             00:37:49      0:0 
5688394.bat+      batch         24        1   00:00:03          0   00:00:03   00:37:49      0:0 
5688394.ext+     extern        180        6   00:00:00          3   00:00:00   00:37:49      0:0 
5688394.0      mrchem.x        180       30   00:37:31         29   01:21:47   00:37:35      0:0 

Memory usage stats:
JobID            MaxRSS MaxRSSTask     AveRSS MaxPages   MaxPagesTask   AvePages 
------------ ---------- ---------- ---------- -------- -------------- ---------- 
5688394                                                                          
5688394.bat+     19332K          0     19332K        0              0          0 
5688394.ext+          0          3          0        0              3          0 
5688394.0     11540180K         29 5107960627        0             22          0 

Disk usage stats:
JobID         MaxDiskRead MaxDiskReadTask    AveDiskRead MaxDiskWrite MaxDiskWriteTask   AveDiskWrite 
------------ ------------ --------------- -------------- ------------ ---------------- -------------- 
5688394                                                                                               
5688394.bat+       16.90M               0         16.90M        0.03M                0          0.03M 
5688394.ext+        0.00M               3          0.00M            0                3              0 
5688394.0           6.75M               0          6.73M      121.93M                0         59.54M 

Job 5688394 completed at Fri May 20 11:08:17 CEST 2022
