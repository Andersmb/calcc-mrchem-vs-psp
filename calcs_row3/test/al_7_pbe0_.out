                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         HEAD                                     ***
***         Git commit hash    e7d4b230bb6df8495b9a                     ***
***         Git commit author  Stig Rune Jensen                         ***
***         Git commit date    Sat Mar 19 14:14:51 2022 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :        (1 bank)                              30
 OpenMP threads          :                                               6
 Total cores             :                                             175
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : MPI/OpenMP (6 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 9
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               2
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Al            :        0.000000        0.911654        0.000000
    1       F            :        0.000000       -0.647366       -2.700310
    2       F            :        0.000000       -0.647366        2.700310
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-04
 Screening               : 1.20000e+01 StdDev
 Restricted              : False
 Functional              : LDA (SVWN5)
 AO basis                : 3-21G
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                              16
 Beta electrons          :                                              15
 Total electrons         :                                              31
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                              1.000000000944e+00
    1    1    a          :                              9.999999997212e-01
    2    1    a          :                              9.999999772979e-01
    3    1    a          :                              9.999999577185e-01
    4    1    a          :                              1.000000001156e+00
    5    1    a          :                              9.999999998584e-01
    6    1    a          :                              9.999999992242e-01
    7    1    a          :                              1.000002457223e+00
    8    1    a          :                              1.000000005221e+00
    9    1    a          :                              1.000000005439e+00
   10    1    a          :                              9.999990805378e-01
   11    1    a          :                              9.999999888122e-01
   12    1    a          :                              9.999999555366e-01
   13    1    a          :                              1.000000745400e+00
   14    1    a          :                              1.000000062916e+00
   15    1    a          :                              1.000000258052e+00
   16    1    b          :                              1.000000063866e+00
   17    1    b          :                              9.999967522565e-01
   18    1    b          :                              9.999956910998e-01
   19    1    b          :                              9.999999578448e-01
   20    1    b          :                              9.999999992181e-01
   21    1    b          :                              9.999999994110e-01
   22    1    b          :                              9.999999998415e-01
   23    1    b          :                              9.999999966118e-01
   24    1    b          :                              1.000000002825e+00
   25    1    b          :                              1.000000010803e+00
   26    1    b          :                              9.999999800368e-01
   27    1    b          :                              1.000000000639e+00
   28    1    b          :                              9.999999785375e-01
   29    1    b          :                              9.999999914689e-01
   30    1    b          :                              9.999999941747e-01
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE0)
 Relativity              : None
 Precision               : 1.00000e-04
 Localization            : On
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                438.809031966295
 E-N energy              :            (au)              -1233.693404637899
 Coulomb energy          :            (au)                305.043747887042
 Exchange energy         :            (au)                 -9.653603145030
 X-C energy              :            (au)                -29.941335595370
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                 90.045253098087
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -529.435563524963
 Nuclear energy          :            (au)                 90.045253098087
---------------------------------------------------------------------------
 Total energy            :            (au)             -4.393903104269e+02
                         :      (kcal/mol)             -2.757215803797e+05
                         :        (kJ/mol)             -1.153619093057e+06
                         :            (eV)             -1.195641892426e+04
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -55.903111582700
    1    1    a          :            (au)                -24.430750172781
    2    1    a          :            (au)                -24.280201926409
    3    1    a          :            (au)                 -4.268905167879
    4    1    a          :            (au)                 -2.796923943519
    5    1    a          :            (au)                 -2.795227122430
    6    1    a          :            (au)                 -2.792391701473
    7    1    a          :            (au)                 -1.098422056117
    8    1    a          :            (au)                 -1.025426627777
    9    1    a          :            (au)                 -0.417166460143
   10    1    a          :            (au)                 -0.337700065169
   11    1    a          :            (au)                 -0.336527756934
   12    1    a          :            (au)                 -0.329637152171
   13    1    a          :            (au)                 -0.259621840683
   14    1    a          :            (au)                 -0.252721750516
   15    1    a          :            (au)                 -0.207747677709
   16    1    b          :            (au)                -55.899472221775
   17    1    b          :            (au)                -24.425673855230
   18    1    b          :            (au)                -24.275731677006
   19    1    b          :            (au)                 -4.261781502023
   20    1    b          :            (au)                 -2.792141545392
   21    1    b          :            (au)                 -2.788407634244
   22    1    b          :            (au)                 -2.787990547380
   23    1    b          :            (au)                 -1.089880804241
   24    1    b          :            (au)                 -1.018700690375
   25    1    b          :            (au)                 -0.390454162810
   26    1    b          :            (au)                 -0.331888457643
   27    1    b          :            (au)                 -0.327383170125
   28    1    b          :            (au)                 -0.300674569524
   29    1    b          :            (au)                 -0.250019777226
   30    1    b          :            (au)                 -0.249554133045
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -242.722237752448
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (PBE0)
 Relativity              : None
 Checkpointing           : Off
 Max iterations          : 200
 KAIN solver             : 10
 Localization            : First two iterations
 Diagonalization         : Off
 Start precision         : 1.00000e-06
 Final precision         : 1.00000e-06
 Helmholtz precision     : Dynamic
 Energy threshold        : 1.00000e-06
 Orbital threshold       : 5.00000e-05
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          5.567764e+00        -439.390310426875         -4.393903e+02
    1          4.110120e-01        -441.905146837285         -2.514836e+00
    2          1.865376e-01        -442.013343026519         -1.081962e-01
    3          1.149458e-01        -442.031288533318         -1.794551e-02
    4          5.610405e-02        -442.038721421563         -7.432888e-03
    5          4.090858e-02        -442.041342844646         -2.621423e-03
    6          1.720876e-02        -442.041679346232         -3.365016e-04
    7          1.061917e-02        -442.041814591618         -1.352454e-04
    8          5.855327e-03        -442.041851311954         -3.672034e-05
    9          3.497772e-03        -442.041861777056         -1.046510e-05
   10          1.972952e-03        -442.041865887833         -4.110776e-06
   11          9.471485e-04        -442.041867065246         -1.177413e-06
   12          5.461569e-04        -442.041867493640         -4.283950e-07
   13          2.565602e-04        -442.041867589625         -9.598477e-08
   14          1.181182e-04        -442.041867610819         -2.119407e-08
---------------------------------------------------------------------------
                      SCF converged in 14 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               2
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Al            :        0.000000        0.911654        0.000000
    1       F            :        0.000000       -0.647366       -2.700310
    2       F            :        0.000000       -0.647366        2.700310
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                441.073967107100
 E-N energy              :            (au)              -1237.206936479410
 Coulomb energy          :            (au)                303.436121539547
 Exchange energy         :            (au)                 -9.591513891942
 X-C energy              :            (au)                -29.798758984203
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                 90.045253098087
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -532.087120708907
 Nuclear energy          :            (au)                 90.045253098087
---------------------------------------------------------------------------
 Total energy            :            (au)             -4.420418676108e+02
                         :      (kcal/mol)             -2.773854576202e+05
                         :        (kJ/mol)             -1.160580755436e+06
                         :            (eV)             -1.202857146778e+04
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -56.203967217447
    1    1    a          :            (au)                -24.745968472483
    2    1    a          :            (au)                -24.745965570107
    3    1    a          :            (au)                 -4.231516423256
    4    1    a          :            (au)                 -2.763775259391
    5    1    a          :            (au)                 -2.761271966623
    6    1    a          :            (au)                 -2.758168009899
    7    1    a          :            (au)                 -1.195608375106
    8    1    a          :            (au)                 -1.188170871243
    9    1    a          :            (au)                 -0.511269653033
   10    1    a          :            (au)                 -0.476225200014
   11    1    a          :            (au)                 -0.449651238419
   12    1    a          :            (au)                 -0.446610029282
   13    1    a          :            (au)                 -0.433684728103
   14    1    a          :            (au)                 -0.427653517121
   15    1    a          :            (au)                 -0.260164522503
   16    1    b          :            (au)                -56.200662625221
   17    1    b          :            (au)                -24.743853062384
   18    1    b          :            (au)                -24.743850994292
   19    1    b          :            (au)                 -4.225637208923
   20    1    b          :            (au)                 -2.759743926076
   21    1    b          :            (au)                 -2.754719727841
   22    1    b          :            (au)                 -2.753763319060
   23    1    b          :            (au)                 -1.190276419763
   24    1    b          :            (au)                 -1.182844535886
   25    1    b          :            (au)                 -0.500122677218
   26    1    b          :            (au)                 -0.465090436537
   27    1    b          :            (au)                 -0.444710743639
   28    1    b          :            (au)                 -0.444561453256
   29    1    b          :            (au)                 -0.432845206984
   30    1    b          :            (au)                 -0.425389270174
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -246.867742661285
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :       -0.000158        0.519674        0.000021
 Magnitude               :            (au)                        0.519674
                         :         (Debye)                        1.320879
---------------------------------------------------------------------------
 Nuclear vector          :        0.000000        0.198920        0.000000
 Magnitude               :            (au)                        0.198920
                         :         (Debye)                        0.505604
---------------------------------------------------------------------------
 Total vector            :       -0.000158        0.718594        0.000021
 Magnitude               :            (au)                        0.718594
                         :         (Debye)                        1.826483
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h 37m 32s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
